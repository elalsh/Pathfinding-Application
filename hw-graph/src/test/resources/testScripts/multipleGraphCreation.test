# Tests multiple graph creation

# Create a graph
CreateGraph graph1

# Add nodes A to graph1
AddNode graph1 A

# Create another graph
CreateGraph graph2

AddNode graph2 B
AddNode graph2 C

AddNode graph1 D

# Print the nodes in the graph1 and graph2
ListNodes graph1
ListNodes graph2