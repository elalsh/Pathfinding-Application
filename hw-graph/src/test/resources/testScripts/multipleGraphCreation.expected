# Tests multiple graph creation

# Create a graph
created graph graph1

# Add nodes A to graph1
added node A to graph1

# Create another graph
created graph graph2

added node B to graph2
added node C to graph2

added node D to graph1

# Print the nodes in the graph1 and graph2
graph1 contains: A D
graph2 contains: B C